transport.go

```diff
@@ -3,11 +3,13 @@ package libp2ptls
 import (
 	"context"
 	"crypto/tls"
+	"errors"
 	"net"
 	"os"
 
 	cs "github.com/libp2p/go-conn-security"
 	ci "github.com/libp2p/go-libp2p-crypto"
+	ic "github.com/libp2p/go-libp2p-crypto"
```

Comment:
Duplicate import.

Reply:
Why is this even allowed in Go... Fixed.